packages/kubernetes/template/script.sh

```diff
@@ -50,8 +50,16 @@ function generate_version_directory() {
     if [ "$minor" -ge 20 ]; then
         criToolsVersion=$(curl -s https://api.github.com/repos/kubernetes-sigs/cri-tools/releases | \
             grep '"tag_name": ' | \
-            grep -Eo "1\.${minor}\.[0-9]+" | \
+            ( grep -Eoq "1\.${minor}\.[0-9]+" || true ) | \
             head -1)
+
+        # Kubernetes 1.23 doesn't have a version fo criTools as of 2021-12-13
```

Comment:
Does this also work for 1.21 and 1.22?

Reply:
Yes, job only recently broke. Critools has 1.21 and 1.22 versions.